test/controllers/nonsipp/memberpensionpayments/MemberPensionPaymentsListControllerSpec.scala

```diff
@@ -102,4 +128,85 @@ class MemberPensionPaymentsListControllerSpec extends ControllerBaseSpec {
     act.like(invalidForm(onSubmit, userAnswers))
     act.like(journeyRecoveryPage(onSubmit).updateName("onSubmit" + _))
   }
+
+  "MemberPensionPaymentsListController in view only mode" - {
+
+    val currentUserAnswers = userAnswers
+      .unsafeSet(FbVersionPage(srn), "002")
+      .unsafeSet(CompilationOrSubmissionDatePage(srn), submissionDateTwo)
```

Comment:
Is it ok not to have member details in current/previous user answers?

Reply:
I'm already reusing userAnswers which has memberDetails set in class scope. I think it should be alright unless you think different.

Reply:
Sorry, I missed out that detail :) It is all right